src/__tests__/Transposit.test.ts

```diff
@@ -358,4 +358,94 @@ describe("Transposit", () => {
       expect(e.message).toEqual(INTERNAL_ERROR_MESSAGE);
     }
   });
+
+  it("deserializes good json in 2XX response", async () => {
+    expect.assertions(1);
+
+    const expected = { everything: "is", a: "okay!" };
+    (window.fetch as jest.Mock).mockReturnValueOnce(
+      Promise.resolve(new Response(JSON.stringify(expected), { status: 200 })),
+    );
+
+    const transposit = new Transposit(BACKEND_ORIGIN);
+    const actual = await transposit.makeCallJson<{}>("GET", "/endpoint");
+
+    expect(actual).toEqual(expected);
+  });
+
+  it("invalid json throws error", async () => {
+    expect.assertions(1);
+
+    const response = new Response("totaljunk", { status: 200 });
+    (window.fetch as jest.Mock).mockReturnValueOnce(Promise.resolve(response));
+
+    const transposit = new Transposit(BACKEND_ORIGIN);
+    return expect(
+      transposit.makeCallJson<{}>("GET", "/endpoint"),
+    ).rejects.toThrow(new APIError("Failed to read response body", response));
+  });
+
+  it("stash get sends correct request", async () => {
```

Comment:
totally optional, but it might be nice to group these three stash tests together under a nested `describe()` block, so you'd have something like
```
describe("stash", () {
  it("get sends correct request", ...);
   ...
});
```
In general, you can nest describe()'s as much as you want, which can sometimes be nice for grouping similar tests together.